code/modules/clothing/neck/_neck.dm

```diff
@@ -187,6 +187,11 @@
 	item_color = "petcollar"
 	var/tagname = null
 
+/obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
+	if(ishuman(M))
```

Comment:
```
if(ishuman(M))
	var/mob/living/carbon/human/H = M
	if(human.dna?.species?.id == "human")
		return FALSE
```

Reply:
Wait, that's valid syntax now? Oh fuck, that'll let us condense a lot of code.

Reply:
also no, this is dumb and you know it